pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -186,3 +198,39 @@ func clusterOwnerRef(clusterName, clusterID string) metav1.OwnerReference {
 		BlockOwnerDeletion: &blockOwner,
 	}
 }
+
+func generateCrashEnvVar(v cephver.CephVersion) corev1.EnvVar {
+	val := fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s -n client.admin", keyring.VolumeMount().AdminKeyringFilePath())
```

Comment:
Is the `client.admin` username not included in the admin keyring file?

Reply:
Yes, but it's not sufficient, well, in this case, it is since `client.admin` is the default user. So I could remove it.